### ISSUE TITLE
XML lib dependencies

### ISSUE THREAD

**Issue by tmorton** [outoftime/noaa#1]
The most recent update changed the gem dependency from libxml-ruby to nokogiri, but the code still depends on libxml-ruby (see NOAA::HttpService).  As a result, I think the version on rubygems is broken:

tmm-2:~ tim$ noaa-update-stations 
Downloading station list from NOAA...
/Users/tim/.rvm/gems/ruby-1.8.7-p174/gems/noaa-0.2.4/bin/../lib/noaa/http_service.rb:16:in `get_station_list': uninitialized constant NOAA::HttpService::LibXML (NameError)
    from /Users/tim/.rvm/gems/ruby-1.8.7-p174/gems/noaa-0.2.4/bin/noaa-update-stations:8
    from /Users/tim/.rvm/gems/ruby-1.8.7-p174/bin/noaa-update-stations:19:in`load'
    from /Users/tim/.rvm/gems/ruby-1.8.7-p174/bin/noaa-update-stations:19

Also, I'm wondering about the versions of this gem.  RubyGems.org has 0.2.4, with all of the earlier versions yanked.  Github has 0.2.3.  I can probably fix the gemspec issues myself, but is there a different repo for the gem code?

**Comment by ProTip** [comment]
Yeah... For anybody reading this, just add the following to noaa.rb:

<code>require 'libxml'</code>

This will fix it.  I get the 0.2.2 version when I us gem to install...  Very odd.

**Comment by tmorton** [comment]
You probably have http://gems.github.com in your gem sources list.  Github isn't building gems any more, but they'll still serve the earlier versions.  Rubygems has the 0.2.4 version, which has this bug.
